backend/service/src/universal_codes/entity_filter.rs

```diff
@@ -116,15 +119,47 @@ pub fn dgraph_filter_from_v1_filter(filter: EntitySearchFilter) -> DgraphFilter
                 ..Default::default()
             }
         }),
-        description: filter.description.map(|_desc| DgraphFilterType {
-            regexp: description_regexp,
-            ..Default::default()
-        }),
         category: filter_categories.map(|categories| DgraphFilterType {
             r#in: Some(categories),
             ..Default::default()
         }),
         r#type: filter_type,
+        description: None,
+        or: None,
+        alternative_names: None,
+    };
+
+    DgraphFilter {
+        or: filter.search.map(|search_string| {
+            vec![
+                DgraphFilter {
+                    description: Some(DgraphFilterType {
+                        regexp: Some(format!("/.*{}.*/i", search_string)),
+                        ..Default::default()
+                    }),
+                    ..base_filter.clone()
+                },
+                DgraphFilter {
+                    alternative_names: Some(DgraphFilterType {
+                        regexp: Some(format!("/.*{}.*/i", search_string)),
+                        ..Default::default()
+                    }),
+                    ..base_filter.clone()
+                },
+                DgraphFilter {
+                    code: Some(DgraphFilterType {
+                        eq: Some(search_string),
```

Comment:
Do you think we'd want the exact match or contains for the codes here?

Reply:
I think equal, because we're usually searching for a copy and pasted code.
I think potentially we could do starts with? Like if you want to type in part of the code if you aren't copy and pasting...
But I don't think we want a search for `ace` to pick up `41ace4bf`?

Reply:
Yeah nice, I'll make it starts_with